packages/ui/package.json

```
@@ -22,7 +22,8 @@
   "scripts": {
     "start": "start-storybook -p 6006 -s public",
     "build": "build-storybook -s public",
-    "test": "react-scripts test"
+    "test": "react-scripts test",
+    "lint": "eslint '*/**/*.{js,ts,tsx}' --config ./.eslintrc.js"
```

Are we having any js files? Lets remove the `js` we can add it if we are having any js files in the `ui`. 
```suggestion
    "lint": "eslint '*/**/*.{ts,tsx}' --config ./.eslintrc.js"
```